apps/ledger-live-mobile/src/components/WebPTXPlayer/CustomHandlers.ts

```diff
@@ -98,7 +98,6 @@ export function usePTXCustomHandlers(manifest: WebviewProps["manifest"], account
                     onSuccess(result.operation.id);
                   }
                   setDevice(undefined);
-                  !result.error && navigation.pop();
```

Comment:
Not sure if it's safe to remove this part too, unless it's expected to not move from here on success